src/mame/miltonbradley/vectrex_v.cpp

```diff
@@ -275,7 +275,7 @@ void vectrex_state::video_start()
 
 void vectrex_base_state::multiplexer(int mux)
 {
-	machine().scheduler().timer_set(attotime::from_nsec(ANALOG_DELAY), timer_expired_delegate(FUNC(vectrex_base_state::update_analog), this), mux);
+	machine().scheduler().timer_set(attotime::from_nsec(ANALOG_DELAY), timer_expired_delegate(FUNC(vectrex_base_state::update_analog), this), m_via_out[PORTA]<<9|0x100|mux);
```

Comment:
Please don’t code like your space bar is broken.

Reply:
Sorry about that, pushed a fix. TBH I'm not convinced this is the best way to handle this, I figured vectrex maintainers would rake me over the coals for using magic numbers and jamming values into simple params.